.github/workflows/integration.yml

```diff
@@ -115,45 +119,207 @@ jobs:
+    outputs:
+      kubernetes_version: ${{ steps.k8s-versions.outputs.versions }}
+    steps:
       - name: Install doctl
         uses: digitalocean/action-doctl@v2
         with:
           token: ${{ secrets.DIGITALOCEAN_ACCESS_TOKEN }}
       - id: k8s-versions
         name: Process k8s version input
         run: |
-          if [ -z '${{ inputs.kubernetes_versions }}' ]; then
+          if [ -z '${{ inputs.kubernetes_version }}' ]; then
             echo "Kubernetes versions not specified, determine DO supported versions"
             versions=`doctl kubernetes options versions -o json | jq -ce 'map(.kubernetes_version)'`
           else
-            versions='${{ inputs.kubernetes_versions }}'
+            versions='${{ inputs.kubernetes_version }}'
           fi
           echo $versions
           echo "versions=$versions" >> $GITHUB_OUTPUT
 
-  run-integration-test:
-    name: Run integration test on cluster
-    needs: preprocess-input
+  run-integration-test-scaleway:
+    if: ${{ inputs.cloud_provider == 'scaleway' }}
+    name: Run integration test on scaleway cluster
+    needs: preprocess-scaleway-input
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        kubernetes_version: ${{ fromJSON(needs.preprocess-scaleway-input.outputs.kubernetes_version) }}
+      max-parallel: 5
+    steps:
+      - name: Use Scaleway CLI
+        uses: scaleway/action-scw@v0
+        with:
+          save-config: true
+          export-config: true
+          version: v2.13.0
+          access-key: ${{ secrets.SCW_ACCESS_KEY }}
+          secret-key: ${{ secrets.SCW_SECRET_KEY }}
+          default-project-id: ${{ secrets.SCW_DEFAULT_PROJECT_ID }}
+          default-organization-id: ${{ secrets.SCW_DEFAULT_ORGANIZATION_ID }}
+      - name: Set k8s cluster name
+        run: |
+          echo SCALEWAY_CLUSTER_NAME=i-${{ github.run_id }}-${{ strategy.job-index }} >> $GITHUB_ENV
+          # Cluster name must be no longer than 63 characters
+      - name: Determine exact k8s version
+        run: |
+          echo SCALEWAY_K8s_VERSION=$(scw k8s version list -o json | jq -re 'map(select(.name | startswith("${{ matrix.kubernetes_version }}"))) | .[].name') >> $GITHUB_ENV
+      - name: Create k8s cluster at Scaleway ${{env.SCALEWAY_CLUSTER_NAME}}
+        run: |
+          out=$(scw k8s cluster create name=${{env.SCALEWAY_CLUSTER_NAME}} \
+              project-id=${{ env.SCW_DEFAULT_PROJECT_ID }} \
+              auto-upgrade.enable=false \
+              cni=calico \
+              pools.0.node-type=${{ env.SCALEWAY_NODE_TYPE }} \
+              pools.0.min-size=${{env.SCALEWAY_NODE_POOL_MIN_SIZE}} \
+              pools.0.size=${{env.SCALEWAY_NODE_POOL_MIN_SIZE}} \
+              pools.0.max-size=3 \
+              pools.0.autohealing=true \
+              pools.0.autoscaling=true \
+              pools.0.name=default \
+              version=${{ env.SCALEWAY_K8s_VERSION }} \
+              region=ams1 \
```

Comment:
It seems that the region is incorrect 😬 
```
{"message":"invalid region ams1","error":{},"hint":"Region format should look like XX-XXX (e.g. fr-par)"}
```

See more: https://github.com/redkubes/otomi-core/actions/runs/5609329423

Reply:
It has been fixed, it should now go to nl-ams